app/views/hideouts/index.html.erb

```diff
@@ -69,8 +72,14 @@
         <% end %>
 
       </div>
+
+    </div>
+    <div id="map" style="width: 100%; height: 600px;" data-markers="<%= @markers.to_json %>"></div>
```

Comment:
No need to add the map one more time here. It's a duplicate. The map is right below, in the `aside` tag.